src/components/MultipleAvatars.js

```diff
@@ -16,62 +17,59 @@ const defaultProps = {
     optionIsFocused: false,
 };
 
-const MultipleAvatars = props => (
-    <>
-        {props.avatarImageURLs !== undefined && (
-            <>
+const MultipleAvatars = ({avatarImageURLs, optionIsFocused}) => {
```

Comment:
Had to redo this whole component, it was causing issues when wrapped by the Pressable. Now also defaults to the regular Avatar if only 1 is passed.
SS of issue below:
<img width="494" alt="Screen Shot 2021-03-01 at 6 52 26 PM" src="https://user-images.githubusercontent.com/12075600/109589528-4e8ae600-7abf-11eb-8945-5fe9737f202b.png">